pyproject.toml

```diff
@@ -1,42 +1,67 @@
-[tool.poetry]
-name = "edx-extensions"
+[project]
+name = "open-edx-plugins"
 version = "0.1.0"
 description = "Collection of plugins for extending and enhancing the functionality of the Open edX platform"
-authors = ["MIT Open Learning Engineering"]
+authors = [
+    {name = "MIT Open Learning Engineering"}
+]
 license = "BSD-3-Clause"
+readme = "README.md"
+requires-python = ">=3.11"
+dependencies = [
+    "Django>4.0",
+    "celery>=4.4.7",
+    "djangorestframework>=3.14.0",
+    "edx-django-utils",
+    "edx-drf-extensions>=10.0.0",
+    "edx-opaque-keys",
+    "gitpython>=3.1.37",
+    "python-json-logger>=3.0.0",
+    "sentry-sdk>=2.0.0",
+    "XBlock",
+    "social-auth-core>=4.5.4",
+    "opentelemetry-distro",
+    "opentelemetry-instrumentation-django",
+    "opentelemetry-exporter-richconsole",
+    "opentelemetry-exporter-otlp-proto-http",
+    "openedx-events",
+]
 
-[tool.poetry.dependencies]
-python = ">=3.8"
-Django = ">2.0"
-celery = ">=4.4.7"
-djangorestframework = "^3.14.0"
-edx-django-utils = ">4.0.0"
-edx-drf-extensions = "^10.0.0"
-edx-opaque-keys = "^3.0.0"
-gitpython = "^3.1.37"
-python-json-logger = "^3.0.0"
-sentry-sdk = "^2.0.0"
-XBlock = "*"
-social-auth-core = "^4.5.4"
-opentelemetry-distro = "*"
-opentelemetry-instrumentation-django = "*"
-opentelemetry-exporter-richconsole = "*"
-opentelemetry-exporter-otlp-proto-http = "*"
-openedx-events = "*"
 
-[tool.poetry.group.dev.dependencies]
-pre-commit = "^3.0.0"
-pytest = "^8.0.0"
-twine = "^5.0.0"
-ruff = "^0.12.0"
-pytest-mock = "^3.14.0"
+
+[tool.uv.workspace]
+members = [
+    "src/*"
+]
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[dependency-groups]
+dev = [
+    "pre-commit>=3.0.0",
+    "pytest>=8.0.0",
+    "pytest-cov>=5.0.0",
```

Comment:
I don't think the coverage plugin was actually running before, but now that it is `**/.coverage` needs to be gitignored.